src/content/self-hosted/administration/configuration.mdx

```diff
@@ -157,7 +157,19 @@ auth:
 
 > You can also use a [secret](self-hosted/administration/configuration.mdx/#cloud-provider-secret) to store the sensitive part of these credentials.
 
-The `organization` field is optional. Only members of the organization will be allowed to log in into your Okteto instance. An empty `organization` field permits any user to log in.
+The `organization` field is optional. Only members of the organization will be allowed to log in into your Okteto instance. An empty `organization` field permits any user to log in. 
+
+If you want to provide access to only select users of your organization, you can additionally use the `allowList` key to list the GitHub accounts you want to allow.
+
+```yaml
+auth:
+  github:
+    enabled: true
+    clientId: clientID
+    clientSecret: clientSecret
+    organization: my-org
+    allowList = ["acct1","acct2","acct3"]
```

Comment:
```suggestion
    allowList: ["acct1","acct2","acct3"]
```

Is this the correct syntax for this line item? Noticed after hitting approve 🤦‍♂️

Reply:
Yes, your suggestion is the right syntax to use